examples/backup_vm.py

```diff
@@ -462,12 +478,15 @@ def get_vm_disks(connection, vm_id):
     return disks
 
 
-def get_backup_events(connection, search_id):
+def get_last_backup_event(connection, search_id):
     events_service = connection.system_service().events_service()
+    backup_events = events_service.list(search=str(search_id))
+    if not backup_events:
+        return None
 
-    # Get the backup events arranged from the most recent event to the oldest
-    return [dict(code=event.code, description=event.description)
-            for event in events_service.list(search=str(search_id))]
+    # First event is the most recent.
```

Comment:
`First event` --> `First event in the events list`.
Otherwise together with the following line it sounds  like "First event is last event" :)

Reply:
I can refine the comment.